Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs

```diff
@@ -8,7 +8,18 @@ public abstract partial class SharedPuddleSystem
     [ValidatePrototypeId<ReagentPrototype>]
     private const string Water = "Water";
 
-    public static readonly string[] EvaporationReagents = [Water];
+    private const string FluorosulfuricAcid = "FluorosulfuricAcid"; // Frontier
+    private const string Vomit = "Vomit"; // Frontier
+    private const string InsectBlood = "InsectBloodt"; // Frontier
+    private const string AmmoniaBlood = "AmmoniaBlood"; // Frontier
+    private const string ZombieBlood = "ZombieBlood"; // Frontier
+    private const string Blood = "Blood"; // Frontier
+    private const string Slime = "Slime"; // Frontier
+    private const string CopperBlood = "CopperBlood"; // Frontier
+    private const string Sap = "Sap"; // Frontier
+    private const string JuiceTomato = "JuiceTomato"; // Frontier
```

Comment:
Shouldn't these have the same ValidatePrototypeId attribute added to them in the same way Water does?  Was this excluded for a reason?

For what it's worth, I'm not sure what impact, if any, that has at runtime, and I haven't been able to run the IntegrationTests.

```Csharp
    // Frontier: extend set of evaporating reagents (PR#1407)
    [ValidatePrototypeId<ReagentPrototype>]
    private const string FluorosulfuricAcid = "FluorosulfuricAcid";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Vomit = "Vomit";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string InsectBlood = "InsectBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string AmmoniaBlood = "AmmoniaBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string ZombieBlood = "ZombieBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Blood = "Blood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Slime = "Slime";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string CopperBlood = "CopperBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Sap = "Sap";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string JuiceTomato = "JuiceTomato";
    // End Frontier
```

Reply:
Trying to keep it simple even if its shit since its coming down from Upstream